pkg/storage/bucket/s3/config.go

```diff
@@ -113,6 +119,9 @@ func (cfg *Config) Validate() error {
 			return errInvalidEndpointPrefix
 		}
 	}
+	if !util.StringsContain(supportedStorageClasses, cfg.StorageClass) && (cfg.StorageClass != "") {
```

Comment:
[nit] Extra parentheses are not required 

```suggestion
	if !util.StringsContain(supportedStorageClasses, cfg.StorageClass) && cfg.StorageClass != "" {
```